sippy-ng/src/App.js

```diff
@@ -189,282 +190,291 @@ export default function App(props) {
   let landingPage = ''
   if (fetchError !== '') {
     landingPage = <Alert severity="error">{fetchError}</Alert>
-  } else if (releases.length > 0) {
-    landingPage = <ReleaseOverview key={releases[0]} release={releases[0]} />
+  } else if (releases.releases.length > 0) {
```

Comment:
Not required, but to avoid stack traces, we can protect against null `releases` or null `releases.releases`, like: `if (releases && releases.releases && releases.releases.length > 0)` or `if (releases?.releases?.length) {`

Reply:
Yea that's a fair comment. Will fix

Reply:
Fixed